src/status_im/contexts/wallet/sheets/missing_keypair/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im.contexts.wallet.sheets.missing-keypair.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.sheets.missing-keypair.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [name emoji color] :as _account} keypair]
```

Comment:
You need to de-structure here to get the keypair `type` and navigate the user depending on it.

```clojure

;; screen-name
(case type
:seed
:screen/settings.missing-keypair.import-seed-phrase
:key
:screen/settings.missing-keypair-import-private-key
nil)


(rf/dispatch [:navigate-to screen-name keypair])
```